Dockerfile

```
@@ -0,0 +1,7 @@
+FROM python:3.5.4-alpine
```

Remove -alpine from the image name.
```suggestion
FROM python:3.5.4
```